js/ord-schema/proto/reaction_pb.js

```diff
@@ -4814,7 +4814,9 @@ proto.ord.ReactionRole.ReactionRoleType = {
   WORKUP: 5,
   INTERNAL_STANDARD: 6,
   AUTHENTIC_STANDARD: 7,
-  PRODUCT: 8
```

Comment:
I think you need to pull in the latest changes from `main` to make sure the proto updates won't conflict with some recent changes upstream.

Reply:
ah ok, false alarm; looks like you already merged everything relevant here